common/app/model/dotcomrendering/DotcomRenderingFrontsModel.scala

```diff
@@ -0,0 +1,72 @@
+package model.dotcomrendering
+
+import common.Edition
+import common.Maps.RichMap
+import common.commercial.EditionCommercialProperties
+import conf.Configuration
+import experiments.ActiveExperiments
+import model.PressedPage
+import navigation.Nav
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.mvc.RequestHeader
+import views.support.{CamelCase, JavaScriptPage}
+
+case class DotcomRenderingFrontsModel(
```

Comment:
At the moment we have 
- DotcomRenderingDataModel
- DotcomBlocksRenderingDataModel
So if we follow that pattern this would be 
- DotcomFrontsRenderingDataModel
But we could otherwise update the two existing class names, so long as it's consistent 🙏